content/md/articles/cookbooks/cli_build_projects.md

```diff
@@ -772,3 +772,48 @@ and calls `jar` which uses
 [both `with-dir` and `with-project-root`](https://github.com/polyfy/polylith/blob/9053b190d5f3b0680ac4fe5c5f1851f7c0d40830/build.clj#L146-L147)
 to set the project root while performing `tools.deps` and `tools.build`
 operations.
+
+## Including java code in a Clojure project
```

Comment:
Can you make sure Java (and Clojure) are capitalized where they refer to the language? There are several places where `java` or `clojure` is used that should be capitalized.

Reply:
Absolutely! I thought my grammar checker did this for me...